lldb/test/Shell/Watchpoint/LocalVariableWatchpointDisabler.test

```diff
@@ -1,3 +1,10 @@
 # REQUIRES: system-darwin
+# TODO: This test is breaking with my output 
+# reformatting done for Large Watchpoint support,
+# but the lines being output by lldb are identical,
+# by visual inspection.  
+# FileCheck is seeing some difference between them,
+# which I need to get to the bottom of.
```

Comment:
A trick I used to do for python doctests was `str.replace(" ", "?")`. Then you could see the trailing spaces. Might work here if you have that or extra newlines.